index.d.ts

```diff
@@ -103,6 +103,17 @@ export type FastifySwaggerInitOAuthOptions = Partial<{
   usePkceWithAuthorizationCodeGrant: boolean
 }>
 
+type JSONValue =
+    | string
+    | number
```

Comment:
```suggestion
    | null
    | number
```
`null` is a valid type too for a json object ^^

Reply:
ty for spotting that. the trouble with making changes late in the evening is you (me) miss the small details